packages/tadviewer/src/components/CellClickData.tsx

```diff
@@ -0,0 +1,8 @@
+export interface CellClickData {
+  readonly value: string;
+  readonly cell: { row: number; col: number };
```

Comment:
Can `row` and `col` be made `readonly`, or does that cause a problem somewhere?

Reply:
🤦 No issues, I just missed it